lib/widget/button/black_button.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:berikan/common/style.dart';
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class BlackButton extends StatelessWidget {
+  final String text;
+  final Function() onPressed;
+  const BlackButton({Key? key, required this.text, required this.onPressed}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Align(
+      child: SizedBox(
+        width: 300,
+        height: 72,
+        child: ElevatedButton(
+          onPressed: onPressed,
+          child: Text(
```

Comment:
Daripada menggunakan `Align` dan `SizedBox` kita bisa memanfaatkan `Padding` di dalam `ElevatedButton`, kemudian  taruh `Text` di dalam `Padding` tersebut.  
Sekali lagi, design dalam Figma hanyalah design kasar.  Mas  @Gdcr7  bisa ubah implementasinya agar terlihat lebih enak.
```suggestion
    return ElevatedButton(
      onPressed: onPressed,
      child: Padding(
        padding: EdgeInsets.symmetric(vertical: 16.0),
        child: Text(
```

Reply:
oke, layoutnya akan ku adjust sekarang